R/simulation-settings.R

```diff
@@ -43,6 +46,19 @@ SimulationSettings <- R6::R6Class(
       }
     },
 
+    #' @field maxSimulationsPerCore  Scale factor used in a parallel simulation.  The product of this scale factor and the number of allowable cores (allowedCores) sets the maximum number of simulations that may be run on one core.
+    maxSimulationsPerCore = function(value) {
+      if (missing(value)) {
+        private$.maxSimulationsPerCore
+      } else {
+        if (!is.null(value)) {

```

Comment:
I would not check for null. It's an error to set a value explicitelu to Null no?

Reply:
done